uvc-src/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "uvc-src"
-version = "0.2.1"
+version = "0.2.2"
 authors = ["Magnus Ulimoen <flymagnus@gmail.com>"]
 edition = "2018"
```

Comment:
I could do this myself, but would be great if you could turn this to 2021 edition :p

Reply:
done.